src/components/dashboard/sidebar/CouponCodesSummaryCard.jsx

```diff
@@ -80,7 +80,7 @@ const CouponCodesSummaryCard = ({
         : (
           <SidebarCard
             title={(
-              <div className="d-flex align-items-start justify-content-between">
+              <div className="d-flex align-items-center justify-content-between">
```

Comment:
Hm, shouldn't the positioning of the badge/title be consistent between subscription/codes/learner credit? As is, its aligned center for codes but `align-items-start` for subscription or learner credit. Can we make these all consistent in how they behave?